app/src/app/search/components/search-result-pagination.tsx

```diff
@@ -0,0 +1,50 @@
+import { Pagination, PaginationContent,
+    PaginationItem,
+    PaginationNext,
+    PaginationPrevious, PaginationFirst, PaginationLast } from "@/components/ui/pagination";
```

Comment:
It looks like the code needs to be auto formatted to get correct indentations / code style